wix/BuildSetupTranslationTransform.cmd

```diff
@@ -13,12 +13,12 @@ SET LANGIDS=%LANGIDS%,%LANGID%
 
 ECHO Building setup translation for culture "%1" with LangID "%2"...
 REM Build with extra Source Code feature (needs work)
```

Comment:
Same comment here:

I updated the code in the comments below to be in the style of wix v4 and wix v5, but it seems that these comments have been untouched for a while now. Would it be better to delete them?

Reply:
I would remove if they're not providing clarity

Reply:
These comments contain code that, in theory, someone could uncomment and run to "Build with extra source code." That said, they had not been touched in 6 years and seemed to be out of date anyways. I did my best here to update them to what is currently being used, but it also feels like we can just retire the dead code. If nobody protests then I will remove these comments in the code in a few days

Reply:
yeah I'd remove them

Reply:
Done